Content.Shared/Doors/Systems/SharedAirlockSystem.cs

```diff
@@ -174,8 +180,8 @@ public void SetSafety(AirlockComponent component, bool value)
         component.Safety = value;
     }
 
-    public bool CanChangeState(EntityUid uid, AirlockComponent component)
+    public bool CanChangeState(EntityUid uid, AirlockComponent component, bool isPried = false)
     {
-        return component.Powered && !DoorSystem.IsBolted(uid);
+        return (isPried || component.Powered) && !DoorSystem.IsBolted(uid);
```

Comment:
Assumes that isPried cannot be set if the component was powered to begin with, so this is a nasty invariant chain.